README.md

```diff
@@ -1,7 +1,17 @@
 pystemd
 =======
 
-![Continuous Integration](https://github.com/systemd/pystemd/workflows/Continuous%20Integration/badge.svg?event=push) ![Matrix](https://img.shields.io/matrix/pystemd:matrix.org)
+<div align="center">
+  <p>
+    <a href="https://github.com/systemd/pystemd/actions/workflows/ci.yml">
```

Comment:
This should probably link to https://github.com/systemd/pystemd/actions ?

Reply:
The proper syntax is:
```
[![Continuous Integration](https://github.com/systemd/pystemd/actions/workflows/ci.yml/badge.svg)](https://github.com/systemd/pystemd/actions/workflows/ci.yml)
```

That will be a link and a badge image.

Then do the same thing with the matrix thing

Reply:
yeap... @jaymzh  its right... updating the PR in a sec (just saw the actuall systemd README page)